articles/cognitive-services/Custom-Vision-Service/select-domain.md

```diff
@@ -20,7 +20,8 @@ From the settings tab of your Custom Vision project, you can select a domain for
 
 |Domain|Purpose|
 |---|---|
-|__General__| Optimized for a broad range of image classification tasks. If none of the other domains are appropriate, or you're unsure of which domain to choose, select the General domain. ID: `ee85a74c-405e-4adc-bb47-ffa8ca0c9f31`|
+|__General__| Optimized for better accuracy with comparable inference time as General Domain. Recommend for larger dataset/harder user scenario. This domain will use more training time. ID: `ee85a74c-405e-4adc-bb47-ffa8ca0c9f31`|
+|__General [A1]__| Optimized for a broad range of image classification tasks. If none of the other domains are appropriate, or you're unsure of which domain to choose, select the General domain. ID: `a8e3c40f-fb4a-466f-832a-5e457ae4a344`|
```

Comment:
Is A1 for larger datasets? these descriptions seem to be reversed

Reply:
fixed